assembl/static2/css/components/posts.scss

```diff
@@ -11,6 +11,10 @@
     text-align: center;
   }
 
+  .statistics-container {
+    height: 70%;
```

Comment:
Same comments than before. Check that when you change the window size, the doughnut isn't too big/small, because you use percent...

Reply:
That's the intent